awx/ui/client/src/shared/list-generator/list-actions.partial.html

```diff
@@ -22,7 +22,6 @@
                 ng-show="{{options.ngShow}}"
                 ng-click="$eval(options.ngClick)"
                 toolbar="true"
-                aw-feature="{{options.awFeature}}">
```

Comment:
Need to keep the `>` here.  Lets put it on the line above: `toolbar="true">`

Reply:
Done!